.flake8

```diff
@@ -9,7 +9,9 @@ exclude = venv*,__pycache__,node_modules,bower_components
 ignore = D203,W503
 max-complexity = 13
 max-line-length = 120
-putty-ignore =
-    app/__init__.py, app/main/__init__.py, app/status/__init__.py : +F401
-    app/__init__.py, app/main/__init__.py, app/status/__init__.py, app/create_buyer/__init__.py, app/external/__init__.py : +E402
-    app/__init__.py, app/main/views/*.py : +C901
+per-file-ignores =
+    app/main/__init__.py : F401
+    app/status/__init__.py : F401
+    app/main/__init__.py : E402
```

Comment:
Can we combine this and line 13 with `app/main/__init__.py : E402, F401`? And the same for the status init?